boards/espressif/esp32_devkitc_wroom/doc/index.rst

```diff
@@ -160,7 +160,7 @@ To build the sample application using sysbuild use the command:
 .. zephyr-app-commands::
    :tool: west
    :zephyr-app: samples/hello_world
-   :board: esp_wrover_kit
+   :board: esp32_devkitc_wroom
```

Comment:
Hi @bajec, thanks for fixing this. Would you mind making it as `esp32_devkitc_wroom/esp32/procpu` instead?